src/include/sof/bit.h

```diff
@@ -24,5 +24,9 @@
 #define SET_BIT(b, x)		(((x) & 1) << (b))
 #define SET_BITS(b_hi, b_lo, x)	\
 	(((x) & ((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL)) << (b_lo))
+#define GET_BIT(b, x) \
+	(((x) & (1ULL << (b))) >> (b))
+#define GET_BITS(b_hi, b_lo, x) \
+	(((x) & (((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL) << (b_lo))) >> (b_lo))
```

Comment:
I feel like we could use the MASK macro for GET_BITS...

Reply:
Yep, it seems to be similarly defined.

Reply:
Done!